app/pages/cif/create-project.tsx

```diff
@@ -86,6 +88,16 @@ export function CreateProject({
             />
           </Grid.Col>
         </Grid.Row>
+        <Grid.Row>
+          <Grid.Col>
+            <ProjectFundingStreamForm
```

Comment:
Not sure if the funding stream selector needs to be a form in itself here, or just a form field that needs to be added to our theme